azure-pipelines.yml

```diff
@@ -159,6 +159,7 @@ steps:
     echo "BUILD_REASON:"$BUILD_REASON
     echo "cmd: "$(cmd)
     eval "$(cmd)"
+    eval "$(./.ci/travis.sh git-diff)"
```

Comment:
Do just `./.ci/travis.sh git-diff`

Reply:
Okay, I will keep this in mind. Changes Done